### ISSUE TITLE
Scrollbar-width on firefox 100 on Debian is quite narrow

### ISSUE THREAD

**Issue by luixxiul** [vector-im/element-web#22084]
### Steps to reproduce

1. Open Element with Firefox 100 on Debian
1. Create a room
2. Send messages to have the scrollbar width displayed
3. Open a website with a scroll bar
4. Set `scrollbar-width: thin` for `*` with devtool
5. Make sure the same issue is reproduced there as well

#### What did you expect?

The scroll bar width should have a static width.

#### What happened instead?

It is strangely narrow around 1~2px. When hovering over the scroll bar area the width gets doubled. I suspect this should be same as https://bugzilla.mozilla.org/show_bug.cgi?id=1759762, meaning this would not be a bug of Element Web itself.

https://user-images.githubusercontent.com/3362943/166906854-0a6099c3-de32-40f7-8e39-48df8c373186.mp4

Since it is not unclear when and if this would be fixed soon, should the width be changed from `thin` to `auto`?

### Operating system

Debian

### Browser information

Firefox 100

### URL for webapp

localhost

### Application version

1a0af54ccba5e91b7dee291cc567e5f1feb92050

### Will you send logs?

No

**Comment by elfez** [comment]
Setting `layout.css.scrollbar-width-thin.disabled` in about:config to `true` and reloading the Element tab seems to have made the scrollbar wider for me. I also have `widget.gtk.overlay-scrollbars.enabled` set to `false`.

**Comment by luixxiul** [comment]
This is an upstream issue and there should not be a fix for matrix-react-sdk.
